src/main/java/ch/njol/skript/expressions/ExprVersionString.java

```diff
@@ -88,13 +89,13 @@ public Class<?>[] acceptChange(ChangeMode mode) {
 		return null;
 	}
 
-	@SuppressWarnings("null")
 	@Override
-	public void change(Event e, @Nullable Object[] delta, ChangeMode mode) {
-		if (!(e instanceof PaperServerListPingEvent))
+	@SuppressWarnings("null")
+	public void change(Event event, Object @Nullable [] delta, ChangeMode mode) {
+		if (!(event instanceof PaperServerListPingEvent))
 			return;
-
-		((PaperServerListPingEvent) e).setVersion(((String) delta[0]));
+		if (delta != null)
+			((PaperServerListPingEvent) event).setVersion(((String) delta[0]));
```

Comment:
Null check shouldn't be necessary due to https://github.com/SkriptLang/Skript/blob/fa8a84226126bbb3bdc2cc14c8fc7c84d83b6b23/src/main/java/ch/njol/skript/effects/EffChange.java#L275